bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Target.scala

```diff
@@ -126,13 +125,14 @@ object Target {
   }
 
   object SiderealTarget {
+
+    def apply(name: String, coordinates: Coordinates, magnitudes: List[Magnitude]): SiderealTarget = SiderealTarget(name, coordinates, None, None, None, magnitudes)
```

Comment:
Yes, I did it to simplify the test, I guess I could move it to the tests as you mention